.circleci/config.yml

```diff
@@ -49,6 +49,21 @@ jobs:
           paths:
             - dist
 
+  e2e:
+    <<: *node_executor
+    steps:
+      - ci/pre-setup
+      - node/npm-install
+      - run:
+          name: "Build checkout-js"
+          command: npm run build
+      - run:
+          name: "Install playwright"
+          command: npx playwright install --with-deps
+      - run:
+          name: "Run e2e tests"
+          command: npx playwright test
+
   release:
```

Comment:
I didn't put `e2e` as part of the release workflow. Do we want it there?

Reply:
I guess we should put it there, since we do run tests as part of that too.